ksql-parser/src/main/java/io/confluent/ksql/util/QueryMask.java

```diff
@@ -52,11 +56,42 @@ public static Map<String, String> getMaskedConnectConfig(final Map<String, Strin
       if (ALLOWED_KEYS.contains(e.getKey())) {
         return e.getValue();
       }
-      return MASKED_VALUE;
+      return MASKED_STRING;
     }));
   }
 
-  private static String maskConnectProperties(final String query) {
+  private static String fallbackMasking(final String query) {
+    return fallbackMaskConnectProperties(fallbackMaskValues(query));
+  }
+
+  public static String fallbackMaskValues(final String query) {
```

Comment:
These fallback methods should be private, right? Looks like the only reason this one isn't is because there's a unit test that calls this directly. It'd be nice to keep these private and update the unit test to test a query from `getMaskedStatement()` which is expected to hit this path (otherwise the unit test coverage misses whether `getMaskedStatement()` properly routes to this path or not.